jest.config.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
   transform: {},
+  "testRegex": "./test/.*.test.js$"
```

Comment:
Is this really necessary?

Reply:
Not exactly indeed. :-) The default value of testRegex covers the current skeleton.
I'll revert this line